web/src/styles/lifecycle.css

```
@@ -53,6 +59,1578 @@ html[data-theme="noir-gold"] {
+.inbox-thread-message-time {
+  font-family: var(--font-mono);
+  font-size: 11px;
+  color: var(--text-tertiary);
+}
+
+.inbox-thread-message-body {
+  font-size: 14px;
+  color: var(--text);
+  line-height: 1.5;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.11.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)